hedera-mirror-importer/src/main/resources/db/migration/v1/V1.88.0__add_nft_history_timestamp_ranges.sql

```diff
@@ -0,0 +1,41 @@
+do $$
+declare
+  current record;
+  prior   record;
+begin
+for current in
+      select * from nft
+      union all
+      select * from nft_history
+      order by token_id, serial_number, timestamp_range
+    loop
+        if (prior is null) then
+            prior = current;
+            continue;
+        elsif (current.token_id = prior.token_id and current.serial_number = prior.serial_number
```

Comment:
can simplify this to
```suggestion
        if prior is not null and current.token_id = prior.token_id and current.serial_number = prior.serial_number
```